Automattic/wp-calypso
# Partner Portal: Add license listing data layer

Context:
* Project thread: pbtFPC-Um-p2
* i4 designs: pbtFPC-103-p2

#### Changes proposed in this Pull Request

* Add a live API-driven license listing data layer to the Partner Portal.

#### Testing instructions

* If you do not have a partner key, please contact Infinity for one or you will be unable to view the UI.
* Checkout PR locally, run `yarn && yarn start-jetpack-cloud`.
* Visit http://jetpack.cloud.localhost:3000/partner-portal and select your partner key, if prompted.
* You should be presented with a list of all of your licenses, if any. Pagination will be introduced in a separate patch.
